addons/website_event/views/event_templates_list.xml

```diff
@@ -97,13 +97,11 @@
                                     </button>
                                 </h2>
                                 <div t-attf-class="o_wevent_offcanvas_cat_#{category.name} accordion-collapse collapse">
-                                    <div class="accordion-body pt-0">
+                                    <div class="accordion-body bg-white pt-0">
```

Comment:
This will do for now 👍 
Colors are not consistent but at least the content is readable (I tested by creating with a quite extreme palette).

In `master`we should find a better way to manage offcanvas colors globally.
 
![image](https://github.com/odoo/odoo/assets/8558351/dd76240a-0b7c-4693-a6d1-30311262df59)